src/elfcore.c

```diff
@@ -1785,10 +1836,12 @@ int InternalGetCoreDump(void *frame, int num_threads, pid_t *pids,
   /* Get parent's CPU registers, and user data structure                     */
   {
     #ifndef __mips__
+    #ifndef __aarch64__ // PTRACE_PEEKUSER returns -1 and set errno = 5 (EIO 5 Input/output error)
```

Comment:
What are the ramifications of not setting this **user** value?

Also, did you see this note from the ptrace(2) man page:

>    C library/kernel differences
       At the system call level,  the  PTRACE_PEEKTEXT,  PTRACE_PEEKDATA,  and
       PTRACE_PEEKUSER requests have a different API: they store the result at
       the address specified by the data parameter, and the  return  value  is
       the  error  flag.  The glibc wrapper function provides the API given in
       DESCRIPTION above, with the result  being  returned  via  the  function
       return value.

So, maybe it didn't fail after all?

Reply:
I just realized that this is  `sys_ptrace` from `linux_syscall_support.h`. I tested the results on both platforms.
It returns data on x86 for some offsets and it always returns -1 (EIO) on arm64.
However, I will remove this `#ifdef` and revert the change back to the original code to avoid unforeseen results.